doc/source/running.rst

```diff
@@ -23,7 +23,7 @@ machines.
 The following uwsgi configuration file can be used::
 
   [uwsgi]
-  http = localhost:8041
```

Comment:
Ok, so I guess it's a compile option missing on my install or something. I'll abandon.